internal/updater/changes.go

```diff
@@ -38,6 +39,19 @@ func (cs *ChangeSet) HasChanges() bool {
 	return cs.Component.Version != cs.LastVersion
 }
 
+func generateTagMessage(tags []string) string {
+	if len(tags) == 0 {
+		return ""
+	}
+
+	tagList := strings.Join(tags, ", ")
+	if len(tagList) > 50 {
```

Comment:
why 50? Think this could be less like 10 or even 5

Reply:
its the number of chars. Do you think less than 50 makes sense in that context? For example frasers often has longer tag names as well as multiple tags

Reply:
sorry misread that, then it's fine.